bootstrap/bootstrap.py

```diff
@@ -90,7 +93,38 @@ def validate_host():
             print("For local development, see http://tljh.jupyter.org/en/latest/contributing/dev-setup.html")
         sys.exit(1)
 
+class LoaderPageRequestHandler(SimpleHTTPRequestHandler):
+    def do_GET(self):
+        if self.path == "/logs":
+            with open("/opt/tljh/installer.log", "rb") as log_file:
+                content = log_file.read()
+                self.wfile.write(content)
+        else:
+            self.path = '/index.html'
```

Comment:
For routing, I am thinking:

1. if it's '/logs', do what we're doing now
2. if it's for '/', redirect to '/index.html'
3. If it's for '/index.html', serve the file
4. For anything else, return a '403 Forbidden' or '404 Not Found'

We want to only serve the files we want to expose to the public, since otherwise we might accidentally expose secrets to the whole world. A routing approach that explicitly lists routes allowed helps with this.